### ISSUE TITLE
Review Alt Text on Governance Team Platform Website pages

### ISSUE THREAD

**Issue by chaseakirby** [department-of-veterans-affairs/va.gov-team#40057]
## User Story
As a VFS team member, I want Platform Website to be accessible and have accurate image descriptors.

- Intended Audience: VFS Teams
- Desired Outcome: Accessible Platform Website pages
- Due Date: N/A

## Description
Platform Content Team reviewed our pages and requested that we add alt text to each image.
Chase added basic alt text in the interim to allow for publishing.

## Artifacts Needing Reviewed
- Getting started
  - [x] [Guidance for new teams and team members](https://vfs.atlassian.net/wiki/spaces/GS/pages/2132803700/Guidance+for+new+teams+and+team+members)
- Collaboration cycle
  - [x] [Collaboration Cycle (landing page)](https://vfs.atlassian.net/wiki/spaces/CC/overview?homepageId=1524891996)
  - [x] [Anatomy of a Staging Review Issue Ticket](https://vfs.atlassian.net/wiki/spaces/CC/pages/2060320997/Anatomy+of+a+Staging+Review+Issue+ticket)
- Developer docs / QA and accessibility testing / Accessibility testing
  - [x] [WCAG 2.1 success criteria and foundational testing](https://vfs.atlassian.net/wiki/spaces/pilot/pages/1777893393/WCAG+2.1+success+criteria+and+foundational+testing)
  - [x] [Add accessibility test cases in TestRail](https://vfs.atlassian.net/wiki/spaces/pilot/pages/1601307071/Add+accessibility+test+cases+in+TestRail)
  - [x] [Request support from the VA 508 office
](https://vfs.atlassian.net/wiki/spaces/pilot/pages/1857913061/Request+support+from+the+VA+508+office)

## Tasks
- [x] Review alt text descriptions on Platform Website pages
- [x] Update alt text descriptions if needed

## Acceptance Criteria
- [x] Images on Platform Website are accessible and accurately described.
- [x] Updates have been Peer Reviewed by Governance Team member
- [x] User Story need has been fulfilled

---

## Related Governance Team Resources
_For additional information and guidance to complete tasks, please see links below._
[Documentation Update Guidance for GitHub templates](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2047868956/Guidance+for+updating+GitHub+Issue+templates) <br />
[Documentation Update Guidance for Platform Website pages](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2089091115/Guidance+for+updating+Platform+Website+pages) <br />
[Documentation Update Guidance for Confluence pages](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2089222173/Guidance+for+updating+Confluence+pages) <br />
[Guidelines for Governance Team Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1127219219/Guidelines+for+Team+spaces) <br />
[Guidelines for Platform Products and Services Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1321140361/Guidelines+for+Platform+Products+and+Services+space) <br />
[Description of Platform Website spaces and content](https://vfs.atlassian.net/wiki/spaces/AP/pages/1241579796/Organization) <br />

---
## How to configure this issue
- [ ] **Ensure that story is attached to appropriate Iniative (if not tech debt)** (what body of work is this a part of?)
- [ ] Ensure that any additional labels are added ('tech-debt', 'collaboration-cycle')
- [ ] Ensure that story is in the correct pipeline (Backlog, Planned, In Progress, Review/QA, Closed)
- [ ] Ensure that assignees are correct so you receive all updates/comments on the ticket

**Comment by briandeconinck** [comment]
Updated list of artifacts needing attention. @chaseakirby or @shiragoodman, all of the pages that Governance is responsible for are under Getting Started or Collaboration Cycle -- correct? (Thankfully not a ton of images on those pages.)

**Comment by chaseakirby** [comment]
Recommend reviewing; How to prepare for 508 staging, defect/severity rubric page.
Check all grouped under accessibility heading.
Post launch audit page should be retired.
Jim - Can we do longer alt text descriptions in Confluence. Jim will check with Content team

**Comment by briandeconinck** [comment]
Per the [Edit platform website documentation](https://vfs.atlassian.net/wiki/spaces/AP/pages/2046558209/Edit+Platform+website+documentation) guidance, changes have been made directly to the impacted pages. None of these edits were longer than a paragraph or changed the meaning of the content of the page, they just better described the contents of the images.

**Comment by shiragoodman** [comment]
@briandeconinck thank you. After Platform Content runs the scroll viewport update, can you verify that the alt text changes are live on Platform Website? This ticket can be closed once verified.

**Comment by shiragoodman** [comment]
@briandeconinck Platform Content flagged an issue related to no alt text on the Guidance for new teams and team members page, however I do see alt text. Reopening this until it's resolved.

https://dsva.slack.com/archives/C01AS05BQ79/p1656006205707439?thread_ts=1656005915.465519&cid=C01AS05BQ79

**Comment by shiragoodman** [comment]
ok turns out it was an error! Closing this back out.
